src/popup/pages/MultisigProposalDetails.vue

```diff
@@ -474,7 +476,18 @@ export default defineComponent({
       try {
         const { gaAccountId, txHash, nonce } = activeMultisigAccount.value;
         const rawTx = await fetchTransactionByHash(txHash as string);
-        transaction.value = await sendTx(gaAccountId, (rawTx as any).tx, nonce);
+        if (!rawTx) {
+          throw Error('failed to load a transaction');
+        }
+        transaction.value = JSON.parse(
+          JSON.stringify(
```

Comment:
Why `JSON.parse/JSON.stringify` is used here?

Reply:
In order to convert all the inner `bigint` values into `string`.

Reply:
Why not `.toString()` ?